user/sorghum-lines-genomics-year-2.md

```diff
@@ -0,0 +1,196 @@
+# Sorghum Lines for Genomics Experiment \(Year 2\)
+
+| **accession** | [**taxid**](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=4558) | [**organism common name**](https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=4558) | **subspecific genetic lineage rank** | **subspecific genetic lineage name** | [**ploidy**](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor) | [**number of replicons**](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor) | [**estimated size**](https://phytozome.jgi.doe.gov/pz/portal.html#!info?alias=Org_Sbicolor) | [**source material identifiers**](https://www.genesys-pgr.org/) | **sample name** |
```

Comment:
Which field can be used to cross-reference the cultivars table in bety  (http://terraref.ncsa.illinois.edu/bety/api/beta/cultivars?key=9999999999999999999999999999999999999999&limit=none)

see also https://github.com/terraref/reference-data/issues/65